core/services/ardupilot_manager/mavlink_proxy/main.py

```diff
@@ -1,19 +1,19 @@
 #!/bin/env python
 import argparse
+import sys
 import time
+from pathlib import Path
+
+# Import local library
+sys.path.append(str(Path(__file__).absolute().parent.parent))
```

Comment:
if all things you are importing are in the mavlink_proxy folder, you shouldn't need this...

Reply:
The problem is that besides the main.py, mavlink_proxy is a library, if you want to use it from another main, like ardupilot_manager, it'll fail to include itself.
Calling `ardupilot_manager/main.py` will result in:
```
Traceback (most recent call last):
  File "./main.py", line 7, in <module>
    from mavlink_proxy.Manager import Manager
  File "/home/patrick/git/blue/companion-docker/core/services/ardupilot_manager/mavlink_proxy/Manager.py", line 6, in <module>
    import MAVLinkRouter
ModuleNotFoundError: No module named 'MAVLinkRouter'
```

And pytest will also fail, since it'll be unable to locate the library, the problem is the same related to the use of the library from `ardupilot_manager/main.py`

Reply:
could it be installed as mavlink_proxy library?

Reply:
yes, but for that we'll need to install first and run the test later ?

Reply:
(we discussed this over a call) This file is a tool, and not a core part of the library, so I'm ok with it.